tests/atoms/gaia_apps.js

```diff
@@ -279,10 +279,12 @@ var GaiaApps = {
       };
 
       let displayedApp = GaiaApps.getDisplayedApp();
-      if (displayedApp.manifestURL == manifestURL ||
+      if ((displayedApp.manifestURL == manifestURL &&
+          displayedApp.entryPoint == entryPoint) ||
           (!displayedApp.manifestURL && displayedApp.origin == origin)) {
         console.log('app with origin \'${displayedApp.origin}\' and ' +
-          'manifestURL \'${displayedApp.manifestURL}\' is already running');
+          'manifestURL \'${displayedApp.manifestURL}\' and ' +
```

Comment:
For this line, `adb logcat | grep 'Content JS LOG'` returns 
`I/GeckoConsole( 9414): Content JS LOG: app with origin '${displayedApp.origin}' and manifestURL '${displayedApp.manifestURL}' has launched`

This line was buggy, let's change it to:
```js 
console.log("app with origin '" + displayedApp.origin"' and " +
"manifestURL '" + displayedApp.manifestURL "' and " +
"entryPoint '" + displayedApp.entryPoint+ "' is already running');
```